### DISCUSSION TITLE
Use "native" detection with supported cameras

### DISCUSSION THREAD

**Discussion by svrooij** [blakeblackshear/frigate#13086] [Ideas]
I have several Unifi protect cameras around the house and they support detections on movement. Would it be possible to use those detections as an input for Frigate.

As in don't decode video for detections, only start doing something when Unifi tells you there was movement. And then in Frigate you can set specific zones to detect specific movement in specific zones and do object detection.

This might help in it being far less busy for the cpu?

**Comment by hawkeye217** [comment]
Unfortunately this is not possible. Some cameras might have on-board motion detection or even AI object detection, but there is no consistent standard or API to be able to get that data from the camera into frigate.

**Reply by NickM-27** [reply]
And even if they did, it is very unlikely they would provide enough information for frigates features to work well.

**Comment by Test-subj** [comment]
Sorry to necropost, but I feel like it could potentially be accomplished through home assistant:
Use HA integration for the camera brand to recognize the camera's detection and then have HA tell Frigate to do something.
Both of those are already available at this point but I don't have the brain to figure it out rn.
